client/stories/eventrow.js

```diff
@@ -1,22 +0,0 @@
-import React from 'react';
```

Comment:
[Updated version of this file](https://github.com/codefordenver/encorelink/pull/377/files#diff-bae5a47d39013a0af5f8ee1c1601d7d1)